javascript/packages/orchestrator/src/providers/k8s/kubeClient.ts

```diff
@@ -83,14 +83,18 @@ export class KubeClient extends Client {
 
   async validateAccess(): Promise<boolean> {
     try {
-      const result = await this.runCommand(["cluster-info"], { scoped: false });
+      const result = await this.runCommand(["auth", "whoami"], {
+        scoped: false,
+      });
       return result.exitCode === 0;
     } catch (e) {
       return false;
     }
   }
 
   async createNamespace(): Promise<void> {
+    // skip if ZOMBIE_NAMESPACE is set
+    if (process.env.ZOMBIE_NAMESPACE) return;
```

Comment:
```suggestion
```
Not needed, since is covered by https://github.com/paritytech/zombienet/pull/2006/files#diff-8e98fb9b4dffcbe5d9f32ec1249415fdb33409effb5b6694a33aa16c5f8661a3R224-R227

Reply:
done!